src/usr/isteps/istep10/barreleyeG2_dynamic_bifurcation.C

```diff
@@ -0,0 +1,109 @@
+/* IBM_PROLOG_BEGIN_TAG                                                   */
+/* This is an automatically generated prolog.                             */
+/*                                                                        */
+/* $Source: src/usr/isteps/istep10/barreleyeG2_bifur.C $            */
```

Comment:
We should avoid having files for specific systems.  This actually looks like a pretty generic implementation where the bifurcation is determined by a sensor lookup from the BMC.  Maybe a better name would be "sensor_based_bifurcation"?

Reply:
Good idea, I will change the naming. Thanks